panel/param.py

```diff
@@ -270,11 +267,7 @@ def widget(self, p_name):
         value = getattr(self.object, p_name)
 
         kw = dict(value=value, disabled=p_obj.constant)
-
-        if self.label_formatter is not None:
-            kw['name'] = self.label_formatter(p_name)
-        else:
-            kw['name'] = p_name
+        kw['name'] = p_obj.label
```

Comment:
Just inline that in ``kw = dict(value=value, disabled=p_obj.constant, name=p_obj.label)``

Reply:
Sure.

Reply:
Fixed in ba512b0